packages/cisco_meraki_metrics/manifest.yml

```
@@ -0,0 +1,35 @@
+format_version: 3.2.0
+name: cisco_meraki_metrics
+title: Cisco Meraki Metrics
+version: 0.0.1
```

```suggestion
version: 0.1.0
```

as we are now going with 0.1.0